### PR TITLE
Emergency activity page

### PR DESCRIPTION
**Pull request by samshara**
## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

File: src/views/EmergencyActivities/ActivityDetail/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+.tooltip {
+    display: flex;
+    align-items: center;
+    padding: var(--go-ui-spacing-sm);
+}
+
+
+
+
+
```
Comment (tnagorra) [inline src/views/EmergencyActivities/ActivityDetail/styles.module.css]:
```suggestion
```

File: src/views/EmergencyActivities/ActivityDetail/index.tsx [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+                    >
+                        {strings.showMore}
+                    </Button>
+                )
+            )}
+            headerDescription={(
+                <>
+                    <div className={styles.dates}>
+                        <DateOutput value={project.start_date} className={styles.date} />
+                        <DateOutput value={project.end_date} className={styles.date} />
```
Comment (tnagorra) [inline src/views/EmergencyActivities/ActivityDetail/index.tsx]: Let's break these down into different lines

File: src/components/domain/DistrictMultiCountrySearchMultiSelectInput.tsx
Comment (tnagorra) [inline src/components/domain/DistrictMultiCountrySearchMultiSelectInput.tsx:1]: What is the component supposed to do?

Comment (tnagorra) [reply src/components/domain/DistrictMultiCountrySearchMultiSelectInput.tsx:1]: Feels like Java. :)

File: src/views/EmergencyActivities/index.tsx [hunk 3]
```diff
@@ -1,17 +1,450 @@
...
+                                to="newThreeWActivity"
+                                state={{ activityId: item.id }}
+                                icons={<CopyLineIcon />}
+                            >
+                                {strings.threeWDuplicate}
+                            </DropdownMenuItem>
+                        </>
+                    ),
+                }),
+            ),
```
Comment (tnagorra) [inline src/views/EmergencyActivities/index.tsx]: Let's create a separate Action component

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: src/components/domain/DistrictMultiCountrySearchMultiSelectInput.tsx
Comment (samshara) [reply src/components/domain/DistrictMultiCountrySearchMultiSelectInput.tsx:1]: select districts for multiple countries

**Review by frozenhelium — APPROVED** [review]

**Pull request merged**